hashin.py

```diff
@@ -68,15 +70,38 @@ def run(spec, file, algorithm, verbose=False):
         if verbose:
             _verbose("Latest version for", version)
 
-    # needs to be turned into a list so we know its length
-    hashes = list(get_hashes(data, version, algorithm, verbose=verbose))
+    try:
+        releases = data['releases'][version]
+    except KeyError:
+        raise PackageError('No data found for version {0}'.format(version))
+
+    # Expand all python versions (3.5 -> [3.5, py3, py2.py3, cp35], etc).
+    python_versions = list(chain.from_iterable(expand_python_version(v) for v in python_versions))
```

Comment:
Also, that comment belongs in the doc string of `expand_python_version` instead.